.github/workflows/autodocs-images.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build Images Reference Docs
+on:
+  workflow_dispatch:
+env:
+  WORKDIR: images
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out Destination Repo"
+        uses: actions/checkout@v2
+        with:
+          path: edu
+
+      - name: "Check out Images Monorepo"
+        uses: actions/checkout@v2
+        with:
+          repository: chainguard-images/images
+          path: images
+
+      - name: "Set up workdir"
+        run: mkdir -p "${{ github.workspace }}/${{ env.WORKDIR }}" && chmod 777 "${{ github.workspace }}/${{ env.WORKDIR }}"
+
+      - name: "Update the reference docs for Chainguard Images"
+        uses: chainguard-dev/deved-autodocs@1.0.2
+        with:
+          command: build images
+        env:
+          YAMLDOCS_SOURCE: "${{ github.workspace }}/images/images"
+          YAMLDOCS_OUTPUT: "${{ github.workspace }}/${{ env.WORKDIR }}"
+
+      - name: "Copy updates to main repo"
+        run: |
+          echo "Copying files..." && \
+          cp -R "${{ github.workspace }}/${{ env.WORKDIR }}" "${{ github.workspace }}/edu/content/chainguard/chainguard-images" && \
+          echo "Finished copy"
+
+      - name: Create a PR
+        uses: peter-evans/create-pull-request@v4
+        with:
+          path: "${{ github.workspace }}/edu"
+          commit-message: Update Images Reference
+          title: "[AutoDocs] Update Images Reference Docs"
+          signoff: true
+          labels: |
+            documentation
+            images
+            automated
+          assignees: erikaheidi
+          reviewers: |
+            erikaheidi
```

Comment:
Did you want to add the code owners so this could be a variable?

Reply:
Good point, let me try adding that too.

Reply:
Ok, added the CODEOWNERS file, and removed the reviewers portion since that is not necessary. When this is merged, all PRs will automatically tag @chainguard-dev/deved-team for a review. To enforce the review, I think we'll need to set up branch protection thou.